src/prefect/environments/storage/azure.py

```diff
@@ -39,12 +40,17 @@ def __init__(
         self.flows = dict()  # type: Dict[str, str]
         self._flows = dict()  # type: Dict[str, "prefect.core.flow.Flow"]
 
-        self.connection_string = connection_string or os.getenv("CONNECTION_STRING")
+        self.connection_string = connection_string or os.getenv(
```

Comment:
Can you update the reference to the env var in https://docs.prefect.io/cloud/execution/storage_options.html#azure-blob-storage

Reply:
I'll actually take this opportunity to update all of those docs

Reply:
@joshmeek updated this file